code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -621,6 +703,9 @@
 	if(!.)
 		return
 
+	if(owner.status_flags & INCORPOREAL)
+		return FALSE
```

Comment:
yeah just this everywhere you've done it

Reply:
moved em up